specification/datashare/resource-manager/readme.python.md

```diff
@@ -3,15 +3,34 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
+```yaml $(python) && !$(track2)
```

Comment:
The configuration is out of time, please do not edit readme.python.md. Just keep file same as 'master' branch